nin/dasBoot/LayerManager.js

```diff
@@ -54,6 +54,28 @@ LayerManager.prototype.update = function(frame) {
         relativeFrame = frame - layer.startFrame;
 
     if (li) {
+      if(layer.animationClips) {
+        for(var key in layer.animationClips) {
+          var animationClip = layer.animationClips[key];
+          for(var j = 0; j < animationClip.length; j++) {
+            var keyframe = animationClip[j];
+            if(keyframe.relativeFrame >= relativeFrame) {
+              var from = animationClip[j - 1];
+              var to = keyframe;
+              break;
+            }
+          }
+          if(from) {
+            li[key] = window[to.type](from.value, to.value,
+                                      (relativeFrame - from.relativeFrame) /
+                                      (to.relativeFrame - from.relativeFrame));
+          } else if(to) {
+            li[key] = to.value;
+          } else {
+            li[key] = from.value;
```

Comment:
As I read this, it seems that the only case when we reach this else is when both `from` and `to` are undefined. In that case, `from.value` won't work here.